pkg/engine/artifacts.go

```diff
@@ -211,7 +211,7 @@ func kubernetesAddonSettingsInit(p *api.Properties) []kubernetesComponentFileSpe
 			sourceFile:      "kubernetesmasteraddons-pod-security-policy.yaml",
 			base64Data:      k.GetAddonScript(PodSecurityPolicyAddonName),
 			destinationFile: "pod-security-policy.yaml",
-			isEnabled:       to.Bool(p.OrchestratorProfile.KubernetesConfig.EnablePodSecurityPolicy),
+			isEnabled:       to.Bool(p.OrchestratorProfile.KubernetesConfig.EnablePodSecurityPolicy) || common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.15.0-beta.1"),
```

Comment:
@jackfrancis If the user sets EnablePodSecurityPolicy to false in the api model, then the cluster deployment will fail. Do we want add a validation check and fail if the user sets kube version > 1.15.0-beta.1 and also set enable psp to false?